malwarecage/resources/metakey.py

```diff
@@ -140,6 +140,61 @@ def post(self, type, identifier):
         schema = MetakeyListResponseSchema()
         return schema.dump({"metakeys": metakeys})
 
+    @requires_authorization
+    @requires_capabilities("removing_attributes")
+    def delete(self, type, identifier):
+        """
+        ---
+        summary: Delete object attribute
+        description: |
+            Delete attribute from specified object.
+
+            User must have `removing_attributes` capability.
+        security:
+            - bearerAuth: []
+        tags:
+            - attribute
+        parameters:
+            - in: path
+              name: type
+              schema:
+                type: string
+                enum: [file, config, blob, object]
+              description: Type of object
+            - in: path
+              name: identifier
+              schema:
+                type: string
+              description: Object identifier
+        requestBody:
```

Comment:
requestBody is not recommended for `GET`, `HEAD` and `DELETE` methods.

Discussion in OAS: https://github.com/OAI/OpenAPI-Specification/issues/1801
https://tools.ietf.org/html/rfc7231#section-4.3.5
```
   A payload within a DELETE request message has no defined semantics;
   sending a payload body on a DELETE request might cause some existing
   implementations to reject the request.
```

In that case: query should be used, see `ObjectResource.get` for proper usage.